staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -290,8 +300,6 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	terminating := apiextensionshelpers.IsCRDConditionTrue(crd, apiextensionsv1.Terminating)
-
 	crdInfo, err := r.getOrCreateServingInfoFor(crd.UID, crd.Name)
```

Comment:
We create handlers and track their corresponding SV updates in the `createCustomResourceDefinition` and `updateCustomResourceDefinition` methods. I think in the `ServeHTTP` method we can only get handlers but not create handlers, since we don't schedule SV update here.

This is because we want to make sure the 1-to-1 relationship between a handler and it's corresponding SV update is tracked.

Reply:
We are no longer creating handlers in `createCustomResourceDefinition` or `updateCustomResourceDefinition` as discussed offline.